### PR TITLE
Issue 11 npm publishing

### LINKED ISSUE #11: publishing to npm
Was just curious if there was any interest in [publishing](https://docs.npmjs.com/cli/v6/configuring-npm/package-json) this project to [**npm**](https://www.npmjs.com)?

I think this would be especially helpful for those who might want to depend on this project via a package manager like `npm` or `yarn`, e.g.
```sh
$ npm install simple.css
```

which would then support continued use and upgrades via _package.json_
```json
"dependencies": {
  "simple.css": "^1.0.0"
}
```

Additionally, I'm pretty sure this would also have a knock on effect of making this automatically available via tools like [**unpkg**](https://unpkg.com), so users could also just pop this into a `<link>` tag as well, e.g.
```html
<link rel="stylesheet" href="https://unpkg.com/simple.css/simple.css">
```

----

To implement, the following steps would need to be taken (and I can help by submitting a PR for steps 1 + 2)
1. Add a _package.json_ to the project with minimum needed keys like `name`, `files`, etc
1. Update _README.md_ / website with different options for usage / installation (package manager, unpkg, copy / paste)
1. Publish (by the maintainer) and test 🤞

### PR DESCRIPTION
**Pull request by thescientist13**
Resolves #11 by introducing a _package.json_ to the project so that it could be published to **npm**.  Opened a PR for docs as well - https://github.com/kevquirk/simple.css-site/pull/3

<details>
@kevquirk  👈 👈 
Please note that for `name`, [_simple.css_ is already taken on **npm**](https://www.npmjs.com/package/simple.css) so as fallback, I've just used your GitHub username as a scope.  Alternatively, you could call your project a different name on **npm**.  

Either way, will need to make sure `name` is not already taken on **npm**.
</details>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@kevquirk/simple.css",
+  "version": "1.0.0",
```
Comment (thescientist13) [inline package.json:3]: note: as part of publishing, you'll need to bump the `version` each time.  There is a [CLI for that](https://docs.npmjs.com/cli/v6/commands/npm-version).

Comment (kevquirk) [reply package.json:3]: What about 'simplecss' for the package name?

Comment (thescientist13) [reply package.json:3]: Looks like it should be [available](https://www.npmjs.com/search?q=simplecss).  👍

**Review by thescientist13 — COMMENTED** [review]
Updated the package name to **simplecss** and updated the [docs PR](https://github.com/kevquirk/simple.css-site/pull/3/commits/3dc4902c87b6dcd7111cd526af52d7279db710b6) as well

**Comment by Pinjasaur** [comment]
I see this got merged (🎉) but it looks like it still needs to be _published_ to the registry.

Might be worth automating that in CI via GitHub Actions. Looks like there's something in the marketplace that should be pretty plug'n'play: https://github.com/marketplace/actions/npm-publish

Thoughts?

**Comment by kevquirk** [comment]
@Pinjasaur it is published - https://www.npmjs.com/package/simpledotcss

We had to change the package name because there's a weird similarly algorithm and "simplecss" already exists.

**Comment by thescientist13** [comment]
@kevquirk 
I'll be able to make the changes to https://github.com/kevquirk/simple.css-site/pull/3/ to reflect the new package name tomorrow so that the docs can be updated as well.

**Comment by kevquirk** [comment]
@thescientist13 thanks, I just merged the changes.

**Pull request merged**